packages/fftw3/fftw3.0.7.2/opam

```diff
@@ -15,7 +15,19 @@ build: [
       "--localstatedir=%{lib}%/fftw3/var"
       "--libdir=%{lib}%/fftw3/lib"
       "--includedir=%{lib}%/fftw3/include"
-      "--datarootdir=%{lib}%/fftw3/share"]
+      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
+  ["./configure" 
+      "CFLAGS=-I/usr/local/include"
+      "LDFLAGS=-L/usr/local/lib"
```

Comment:
On Thu, 28 Jan 2016 09:03:49 -0800, Romain Beauxis wrote:
> 
> In packages/fftw3/fftw3.0.7.2/opam:
> 
> > @@ -15,7 +15,19 @@ build: [
> >        "--localstatedir=%{lib}%/fftw3/var"
> >        "--libdir=%{lib}%/fftw3/lib"
> >        "--includedir=%{lib}%/fftw3/include"
> > -      "--datarootdir=%{lib}%/fftw3/share"]
> > +      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
> > +  ["./configure"
> > +      "CFLAGS=-I/usr/local/include"
> > +      "LDFLAGS=-L/usr/local/lib"
> 
> Yeah. I was having a weird path issue with OSX's `clang` but I fixed it locally in fact. Sorry for the noise..

I intend to commit this to fftw3 repo but I'd like to make sure the proposed patch is working on all MacOSX systems and not only yours.  May you confirm that?